index.js

```diff
@@ -0,0 +1,171 @@
+  row2.remove();
+  }
+}
+
+//experience add and clear button 
+//add button
+let flag = 100;
+let experience_btn = document.getElementById("add_experience");
+let experience_block = document.getElementById("experience-data-1");
+
+experience_btn.addEventListener("click", () => {
+  let experienceDiv = document.createElement("div");
+  flag = flag + 1;
+  let template = `
+    <input type="month" name="experience_start_date" id="experience_start_date" class="experience-sdate-class"/>
+    <input type="month" name="experience_till_date" id="experience_till_date" class="experience-tdate-class"/>
+    <input type="text" name="company name" id="company_name" class="company-name" placeholder="enter company name"/>
+    <button id="${flag}" onclick="removeExperience(this)">Clear</button>
+    `;
+  experienceDiv.id = `experience-data-${flag}`;
+  experienceDiv.insertAdjacentHTML('beforeend', template);
+  experience_block.appendChild(experienceDiv);
+
+})
+
+//clear button
+document.getElementById("100").style.display = "none";
+function removeExperience(button) {
+  if (button.id !== 100) {
+    let num = button.id;
+    let row = document.getElementById('experience-data-' + num);
+    row.remove();
+  }
+}
+
+//certification add and clear button 
+let count = 1;
+let certificate_btn = document.getElementById("add_certificate");
+let certificate_block = document.getElementById("certificate-data-1");
+
+certificate_btn.addEventListener("click", () => {
+  let certificate_Div = document.createElement("div");
+  count = count + 1;
+  let template = `
+        <input type="date" name="Certification_date" id="Certification_date" class="certificate-date-class"/>
+        <input type="text" name="Certification_feild" id="Certification_feild" class="certificate-field-class"/>
+        <input type="text" name="Certification_authority" id="Certification_authority" class="certificate-auth-class"/>
+        <button id="${count}" onclick="removeCertificate(this)">Clear</button>
+    `;
+  certificate_Div.id = `certificate-data-${count}`;
+  certificate_Div.insertAdjacentHTML('beforeend', template);
+  certificate_block.appendChild(certificate_Div);
+})
+
+document.getElementById("1").style.display = "none";
+function removeCertificate(button) {
+  if (button.id != 1) {
+    let num = button.id;
+    let row = document.getElementById('certificate-data-' + num);
+    row.remove();
+  }
+}
+
+
+//template button
+let selected = 0;
+let template_btn1 = document.getElementById("t1");
+template_btn1.addEventListener("click",() => {
+  selected=1;
+  template_btn1.style.backgroundColor = "black";
+  template_btn2.style.backgroundColor = "buttonface";
+});
+let template_btn2 = document.getElementById("t2");
+template_btn2.addEventListener('click',() => {
+  selected=2;
+  template_btn1.style.backgroundColor = "buttonface";
+  template_btn2.style.backgroundColor = "black";
+});
+
+
+//generate button
+let bool=true;
```

Comment:
change this variable name 
For example, if I'm creating a variable to perform the check for the button so I can declare a variable with the name: "isButtonVisible"